### ISSUE TITLE
API Array POST Bug

### ISSUE THREAD

**Issue by cleanspacedev** [FlutterFlow/flutterflow-issues#347]
Issue tracker is **ONLY** used for reporting bugs. New feature suggestions and questions should be discussed on Community or submitted through our user feedback form.

Your issue may already be reported! Please search in the [issue tracker](../) before creating one.

Please **thumbs up** this issue if you have also experienced it. You may also add more information if there is something relevant that was not mentioned. However, please refrain from comments that are not constructive, like "I have this problem too", etc.

## Expected behavior (required)

Send request as an Array.

## Current behavior (required)

Even when the List option is set to True it still sends it as an single value and as a list. 
## To Reproduce (required)

* Version of FlutterFlow used: 3.0
* Platform: Web
* Browser name and version:  Chrome Latest Version
* Operating system and version: Desktop

**Comment by cleanspacedev** [comment]
Any update on this? Do you need more examples?

**Comment by victoriahuang1** [comment]
Hi @cleanspacedev , thanks for the report. Does this API all work if you specify `["file_1MPGw9EVknDkxqHqAkiyPKSK"]` instead of `file_1MPGw9EVknDkxqHqAkiyPKSK` as the value for `fileID`?

**Comment by cleanspacedev** [comment]
It makes it a value and looks like  `\["file_1MPGw9EVknDkxqHqAkiyPKSK"]\`

**Comment by cleanspacedev** [comment]
> Hi @cleanspacedev , thanks for the report. Does this API all work if you specify `["file_1MPGw9EVknDkxqHqAkiyPKSK"]` instead of `file_1MPGw9EVknDkxqHqAkiyPKSK` as the value for `fileID`?

It is as if the parser that is being used for x-www-form-urlencoded does not know how to convert array's and  makes the whole inputted value and variable.

**Comment by victoriahuang1** [comment]
Yes, you're correct that there is no string -> array conversion. But can you verify if the POST endpoint can accept `["file_1MPGw9EVknDkxqHqAkiyPKSK"]`? If not, we can add code to handle arrays.

**Comment by cleanspacedev** [comment]
> Yes, you're correct that there is no string -> array conversion. But can you verify if the POST endpoint can accept `["file_1MPGw9EVknDkxqHqAkiyPKSK"]`? If not, we can add code to handle arrays.

Yes, it would except that, but if I add brackets to the input the parser inserts `\["file_1MPGw9EVknDkxqHqAkiyPKSK"]\ `before executing the request.
